client/app/components/menu_mailbox_item.coffee

```diff
@@ -63,6 +63,6 @@ module.exports = React.createClass
                     span className: 'refresh-error', onClick: displayError,
                         i className: 'fa fa-warning', null
 
-
-            if not @props.isMailboxLoading and @props.unread
-                span className: 'badge', @props.unread
+            div className: 'badges',
```

Comment:
Arent you forget the case where an account is creating? That's why I've added the props `isIndexing`.